Source/WebCore/platform/graphics/ca/GraphicsLayerCA.cpp

```diff
@@ -828,6 +828,13 @@ void GraphicsLayerCA::setOpacity(float opacity)
     noteLayerPropertyChanged(OpacityChanged);
 }
 
+void GraphicsLayerCA::setRenderingSuppressed(bool suppressed)
+{
+    GraphicsLayer::setRenderingSuppressed(suppressed);
+    if (!suppressed)
+        setNeedsDisplay();
```

Comment:
Suppression affects all descendant layers, yet this only repaints this layer which doesn't make sense.